src/components/Api.js

```diff
@@ -0,0 +1,81 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  renderResult(res) {
```

Comment:
It's great to have a method for processing the server response, so the logic is not duplicated